src/ppom-controller.ts

```diff
@@ -327,16 +290,13 @@ export class PPOMController extends BaseControllerV2<
   /**
    * Update the PPOM.
    * This function will acquire mutex lock and invoke internal method #updatePPOM.
-   *
-   * @param options - Options.
-   * @param options.updateForAllChains - True is update if required to be done for all chains in cache.
    */
-  async updatePPOM({ updateForAllChains } = { updateForAllChains: true }) {
+  async updatePPOM() {
```

Comment:
why is this method public? Do we intend to use it on either the extension or mobile?

Reply:
Nevermind.. saw that it is being exposed through the message handlers

Reply:
Yeh it is not currently being used outside the controller, we can make it private also. Let me do that.